webview/src/setup/components/App.test.tsx

```diff
@@ -631,7 +631,7 @@ describe('App', () => {
   })
 
   describe('Studio not connected', () => {
-    it('should show buttons which request a token from Studio or add an already creatd one', () => {
+    it('should show buttons which request a token from Studio or add an already created one', () => {
```

Comment:
Spotted a typo that I went ahead and fixed.

Reply:
[streetsidesoftware.code-spell-checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)